examples/util/modelZoo.js

```diff
@@ -838,6 +838,20 @@ const modelZoo = {
     },
     intro: 'This is a lightweight network for the face re-identification scenario that is\
       based on MobileNet V2. The model produces feature vectors which should be close in cosine distance for similar faces and far for different faces.'
+  }],
+
+  speechRecognitionModels: [{
+    modelName: 'wsj_dnn5b (OpenVINO)',
+    format: 'OpenVINO',
+    modelId: 'wsj_dnn5b_openvino',
+    modelSize: '115.6MB',
+    inputSize: [1, 440],
+    outputSize: [1, 3425],
+    modelFile: '../speech_recognition/model/wsj_dnn5b.bin',
+    arkFile: '../speech_recognition/ark/dev93.ark',
+    scoreFile: '../speech_recognition/ark/dev93_scores.ark',
+    intro: 'Inference based on Kaldi* neural networks and speech feature vectors.',
```

Comment:
For history reason, the intro of all models are not well described, e.g. I prefer to let users know this model is /The Wall Street Journal DNN model was prepared using the Kaldi s5 recipe and the Kaldi Nnet (nnet1) framework./

Reply:
Yes, I'll add more details about this model.